// george-library-two/esm2020/lib/george-library-two.component.mjs
import { Component } from '@angular/core';
import * as i0 from "@angular/core";
export class GeorgeLibraryTwoComponent {
    constructor() { }
    ngOnInit() {
    }
}
GeorgeLibraryTwoComponent.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.1", ngImport: i0, type: GeorgeLibraryTwoComponent, deps: [], target: i0.ɵɵFactoryTarget.Component });
GeorgeLibraryTwoComponent.ɵcmp = i0.ɵɵngDeclareComponent({ minVersion: "12.0.0", version: "13.2.1", type: GeorgeLibraryTwoComponent, selector: "lib-george-library-two", ngImport: i0, template: `
    <p>
      george-library-two works!
    </p>
  `, isInline: true });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.1", ngImport: i0, type: GeorgeLibraryTwoComponent, decorators: [{
            type: Component,
            args: [{
                    selector: 'lib-george-library-two',
                    template: `
    <p>
      george-library-two works!
    </p>
  `,
                    styles: []
                }]
        }], ctorParameters: function () { return []; } });
