// george-library-two/esm2020/lib/george-library-two.module.mjs
import { NgModule } from '@angular/core';
import { GeorgeLibraryTwoComponent } from './george-library-two.component';
import * as i0 from "@angular/core";
export class GeorgeLibraryTwoModule {
}
GeorgeLibraryTwoModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.1", ngImport: i0, type: GeorgeLibraryTwoModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
GeorgeLibraryTwoModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.1", ngImport: i0, type: GeorgeLibraryTwoModule, declarations: [GeorgeLibraryTwoComponent], exports: [GeorgeLibraryTwoComponent] });
GeorgeLibraryTwoModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.1", ngImport: i0, type: GeorgeLibraryTwoModule, imports: [[]] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.1", ngImport: i0, type: GeorgeLibraryTwoModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [
                        GeorgeLibraryTwoComponent
                    ],
                    imports: [],
                    exports: [
                        GeorgeLibraryTwoComponent
                    ]
                }]
        }] });
