// george-library-two/fesm2015/george-library-two.mjs
import * as i0 from '@angular/core';
import { Injectable, Component, NgModule } from '@angular/core';

class GeorgeLibraryTwoService {
    constructor() { }
}
GeorgeLibraryTwoService.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.1", ngImport: i0, type: GeorgeLibraryTwoService, deps: [], target: i0.ɵɵFactoryTarget.Injectable });
GeorgeLibraryTwoService.ɵprov = i0.ɵɵngDeclareInjectable({ minVersion: "12.0.0", version: "13.2.1", ngImport: i0, type: GeorgeLibraryTwoService, providedIn: 'root' });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.1", ngImport: i0, type: GeorgeLibraryTwoService, decorators: [{
            type: Injectable,
            args: [{
                    providedIn: 'root'
                }]
        }], ctorParameters: function () { return []; } });

class GeorgeLibraryTwoComponent {
    constructor() { }
    ngOnInit() {
    }
}
GeorgeLibraryTwoComponent.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.1", ngImport: i0, type: GeorgeLibraryTwoComponent, deps: [], target: i0.ɵɵFactoryTarget.Component });
GeorgeLibraryTwoComponent.ɵcmp = i0.ɵɵngDeclareComponent({ minVersion: "12.0.0", version: "13.2.1", type: GeorgeLibraryTwoComponent, selector: "lib-george-library-two", ngImport: i0, template: `
    <p>
      george-library-two works!
    </p>
  `, isInline: true });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.1", ngImport: i0, type: GeorgeLibraryTwoComponent, decorators: [{
            type: Component,
            args: [{
                    selector: 'lib-george-library-two',
                    template: `
    <p>
      george-library-two works!
    </p>
  `,
                    styles: []
                }]
        }], ctorParameters: function () { return []; } });

class GeorgeLibraryTwoModule {
}
GeorgeLibraryTwoModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.1", ngImport: i0, type: GeorgeLibraryTwoModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
GeorgeLibraryTwoModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.1", ngImport: i0, type: GeorgeLibraryTwoModule, declarations: [GeorgeLibraryTwoComponent], exports: [GeorgeLibraryTwoComponent] });
GeorgeLibraryTwoModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.1", ngImport: i0, type: GeorgeLibraryTwoModule, imports: [[]] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.1", ngImport: i0, type: GeorgeLibraryTwoModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [
                        GeorgeLibraryTwoComponent
                    ],
                    imports: [],
                    exports: [
                        GeorgeLibraryTwoComponent
                    ]
                }]
        }] });

/*
 * Public API Surface of george-library-two
 */

/**
 * Generated bundle index. Do not edit.
 */

export { GeorgeLibraryTwoComponent, GeorgeLibraryTwoModule, GeorgeLibraryTwoService };
